tests/scripts/all.sh

```diff
@@ -121,6 +158,17 @@ fi
 #
 # Indicative running times are given for reference.
 
+msg "info: $0 configuration"
+echo "MEMORY: $MEMORY"
+echo "SHORT: $SHORT"
```

Comment:
The patch that removes the `SHORT` option has not been merged for sometime, so for consistency I will leave this here until that happens and if necessary, update either of the PRs as they are merged.